README.md

```diff
@@ -9,7 +9,7 @@ Amnezia is an open-source VPN client, with a key feature that enables you to dep
 ## Features
 
 - Very easy to use - enter your IP address, SSH login, and password, and Amnezia will automatically install VPN docker containers to your server and connect to the VPN.
-- OpenVPN, ShadowSocks, WireGuard, and IKEv2 protocols support.
+- OpenVPN, Shadowsocks, WireGuard, and IKEv2 protocols support.
```

Comment:
According to https://shadowsocks.org/
![image](https://github.com/amnezia-vpn/amnezia-client/assets/25141164/80524d64-e0bf-4122-8307-0b221a93534b)